inference-engine/tests/unit/vpu/base/graph_transformer_tests.cpp

```diff
@@ -146,9 +150,28 @@ void TestModel::createOutputs(std::vector<DataDesc> outputDescs) {
     }
 }
 
+void TestModel::createInputs(int numInputs) {
+    _model->attrs().set<int>("numInputs", numInputs);
+    _inputs.resize(numInputs);
+
+    for (int i = 0; i < numInputs; ++i) {
+        _inputs[i] = _model->addInputData(formatString("Input %d", i), _dataDesc);
+    }
+}
+
+void TestModel::createOutputs(int numOutputs) {
+    _model->attrs().set<int>("numOutputs", numOutputs);
+    _outputs.resize(numOutputs);
+
+    for (int i = 0; i < numOutputs; ++i) {
+        _outputs[i] = _model->addOutputData(formatString("Output %d", i), _dataDesc);
+    }
+}
+
 Stage TestModel::addStage(
         std::initializer_list<InputInfo> curInputInfos,
-        std::initializer_list<OutputInfo> curOutputInfos) {
+        std::initializer_list<OutputInfo> curOutputInfos,
+        bool write_to_dataDesc) {
```

Comment:
You actually don't need this parameter, just do that:
```c++
curOutputs.push_back(_model->addNewData(formatString("Data %d / %d", _stages.size(),
 curOutputs.size()), info.desc == DataDesc() ? _dataDesc : info.desc));
```

If user hasn't specified anything into `intermediate()` function the default one from model will be used, otherwise the desc passed to `intermediate` will be used.